tests/TaskLineRenderer.test.ts

```diff
@@ -279,6 +280,50 @@ describe('task line rendering', () => {
         );
     });
 
+    it.each([
+        // Nominal cases
+        ['tag at the end #todo', 'tag at the end'],
+        ['tag #tag in the middle', 'tag in the middle'],
+        ['#now tag in the beginning', 'tag in the beginning'],
+        ['#important #s a #today #TODO lot #never of tags #now #PERFORMANCE', 'a lot of tags'],
+        ['nested tags #todo/only/today #x/y/z', 'nested tags'],
+        ['tag with hyphens #a-b-c', 'tag with hyphens'],
+
+        // Border cases
+        ['but      keep extra     spaces', 'but      keep extra     spaces'],
+    ])(
+        'should render without tags task with description "%s"',
```

Comment:
1. I was looking at the pattern of the names of the other tests in this file, for visual pattern-matching...

For consistency, this test would ideally begin `renders without tag`...

2. But then I had a look at the related tests - and there are some for how the CSS is or is not affected by the `hide` command.

This took me to:
https://publish.obsidian.md/tasks/Advanced/Styling#Highlight+for+a+Specific+Tag

I experimented with those two snippets. They are in the Tasks-Demo vault, disabled - and called:
    "tasks-plugin-highlight-specific-tag-green-glow"
    "tasks-plugin-highlight-specific-tag-round-red-description"

Hiding the tags prevents the highlighting of the description, that is, from a user perspective it breaks that CSS snippet.

I suggest logging a bug that hiding tags breaks styling based on the tags - and then add a note in these locations to list the limitation, and say something like 'we are tracking it in...'

- https://publish.obsidian.md/tasks/Advanced/Styling#Highlight+for+a+Specific+Tag
- https://publish.obsidian.md/tasks/Queries/Layout

Reply:
OK the seeming breaking of the CSS examples is the last issue.
If you don't have time to deal with it, please do let me know, and I will...

Reply:
Thanks a lot for your time spent on the review and your flexibility to accept this despite being incomplete versus CSS styling.

I'm not proficient with CSS styles so I don't feel like logging a bug, I just wouldn't know what to describe there. Could you please help with that?

Nevertheless I mentioned the issue in the docs (as you suggested), if you submit an issue, could you please update the links and the issue number? The places to mention are [here](https://github.com/obsidian-tasks-group/obsidian-tasks/pull/2076/commits/aa12c3cc48b3240885064744c37565e087f14047).

PS I would be happy to fix the bug next time we pair, that would be really interesting for me.

Reply:
Thanks. Happy to merge...